### ISSUE TITLE
model.train docstring is out of date

### ISSUE THREAD

**Issue by shuttle1987** [persephone-tools/persephone#138]
```
    def train(self, early_stopping_steps=10, min_epochs=30, max_valid_ler=1.0,
              max_train_ler=0.3, max_epochs=100, restore_model_path=None):
        """ Train the model.

            batch_size: The number of utterances in each batch.
            total_size: The number of TIMIT training examples to use.
            num_epochs: The number of times to iterate over all the training
                        examples.
            feat_type:  Is the identifier for the type of features we're using.
                        'mfcc13_d' means MFCCs of 13 dimensions with their first
                        derivatives.
            save_n: Whether to save the model at every n epochs.
            restore_model_path: The path to restore a model from.
        """
```

Currently there's some parameters that are not matching what is in the docstring.

**Comment by shuttle1987** [comment]
This is related to #79

**Comment by oadams** [comment]
Looks like you've fixed this in #147 in commit fe3462a3a2d60cbd261ddbd18767d08fde0b0582

**Comment by shuttle1987** [comment]
It's best to not close these issues until the code is merged. (The GitHub PR mechanism will automatically close the issues referenced in a merged PR, note how the `Closes #...` is handled)
